docs/tutorial.md

```diff
@@ -0,0 +1,82 @@
+# Getting Started with Paper Programs
+
+This tutorial will cover what you'll need to get a Paper Programs space up and running.
+
+## Hardware Required
+
+- A computer
+- A projector - ideally 2000 lumens and 1080p. A 1000 lumens projector can work under some conditions, but the low light needed to see the projections may be too dark for your camera to detect the programs in the space.
+- A webcam - 720p works, but be wary that cameras with poor quality sensors may not work well in all lighting conditions.
+- A color printer - Any color printer will do.
+- A way to mount the projector - This is a tough one, whatever you do, bear in mind that projectors are heavy enough that they need to be connected to something load-bearing like a beam or a stud. Command Strips or screws in drywall will not be enough. (Note that you can project onto a wall and tape your programs to the wall if you just want to try this out, but it's not quite the same).
+- Optional: Depending on how you set up your space, you may need unusually long HDMI and USB cables to connect the camera/projector to the computer.
+- Optional: A solid black rug or mat to project onto. Bear in mind: a projector's whites can only be as bright as the projector's bulb, and its blacks can only be as black as the surface it is projecting onto.
+
+## Optional: Setting up the server locally
+
+For the rest of this tutorial we'll be using paperprograms.org itself, but if you want to run everything locally here's how to get it all set up:
+
+Make sure to have [Node.js](https://nodejs.org) and [PostgreSQL](https://postgresql.org) installed first. Checkout the repo and run in a terminal:
+
+```sh
+npm install
+npm run dev
+```
+
+This will create a new database, and start all necessary processes. Then go to [localhost:3000](http://localhost:3000) to get started.
+
+If you run into issues, make sure your postgres user is configured correctly, and add their credentials to `knexfile.js` if you run into password issues.
```

Comment:
You should be able to specify a password in `DATABASE_URL`, no need to change any code.

Reply:
Ah, except locally I guess, good point. Alright let's keep this. 👍